misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -208,10 +208,12 @@ internal fun HttpServletResponse.headers(): Headers {
 }
 
 internal fun HttpServletRequest.httpUrl(): HttpUrl {
+  //LOL, the joys of shoehorning jetty server http objects into okhttp.
```

Comment:
For websocket, jetty is setting the `requestURL` as the `ws://host/path/` and okhttp is very confused by the `ws` schema. I'm not sure why we're shoehorning servlet requests into `okhttp`, but we're running into this weird issues. 

I just replaced the `ws` for `http` to unblock myself, but I'm not sure if this is a good idea.  Do you have a better suggestion here @swankjesse ? 

Also, I don't think there's anybody using websockets in misk, I wonder if we should deprecate that feature considering we could be using gRPC streams instead.

Reply:
You wanna be careful about not replacing ws elsewhere in the URL. `https://www.cows.com` should not become `https://www.cohttps.com`.

I used `HttpUrl` because it’s a better URL model than java.net.URL and java.net.URI. It’s true that it’s OkHttp’s model, but that’s only ’cause OkHttp doesn’t have a server (yet?).

Reply:
Some background on the horror show of the other URL classes.
https://developer.squareup.com/blog/okhttps-new-url-class/

Reply:
As for deprecating web sockets? I’m mixed on that. There’s no gRPC streams on the web, and web sockets are great for building interactive UIs on the web.

Reply:
good points. Do you have any concern with replacing `ws://` with `http://` ?

Reply:
I was planning to use Misk websockets with Hotwire for a server driven UI demo so I also hope we don't need to deprecate it.